action.yml

```diff
@@ -19,6 +19,9 @@ inputs:
     default: ''
   sha256-checksum: # the expected SHA256 checksum of the devbox binary.
     description: 'Specify an explicit checksum for the devbox binary. For extra security on top of the existing checks in the devbox launch script'
+  configure-nix-access-token: # 'true' or 'false'
```

Comment:
Can we do `disable-nix-access-token` and default it to `false`? (Assuming that most users would like it to be enabled

Reply:
Done, thanks for the review :)